versions/2016-10-31.md

```diff
@@ -898,11 +898,11 @@ Auth:
     IpRangeBlacklist: [<list of ip ranges>]
     SourceVpcWhitelist: [<list of vpc/vpce endpoint ids>]
     SourceVpcBlacklist: [<list of vpc/vpce endpoint ids>]
-```
 
   # For AWS_IAM:
   # DefaultAuthorizer: AWS_IAM
   # InvokeRole: NONE # CALLER_CREDENTIALS by default unless overridden
+  
```

Comment:
I added a new line here to separate the `AWS_IAM` example configuration from the `Authorizers` section.

Reply:
This was fixed in PR: #1214. It will be merged into master in one of our upcoming releases